README.md

```diff
@@ -23,121 +23,142 @@ Sourcebot is a fast code indexing and search tool for your codebases. It is buil
+Before getting started, please make sure you install <a href="https://docs.docker.com/get-started/get-docker/"><img src="https://www.docker.com/favicon.ico" width="16" height="16"> Docker </a>. This is the only dependency required to get started using Sourcebot.
 
-1. Create a `config.json` file and list the repositories you want to index. The JSON schema [index.json](./schemas/index.json) defines the structure of the config file and the available options. For example, if we want to index Sourcebot on its own code, we could use the following config found in `sample-config.json`:
+You can try out Sourcebot on your local machine without cloning the repo by running this simple command:
 
-    ```json
-    {
-        "$schema": "https://raw.githubusercontent.com/TaqlaAI/sourcebot/main/schemas/index.json",
-        "Configs": [
-            {
-                "Type": "github",
-                "GitHubOrg": "TaqlaAI",
-                "Name": "^sourcebot$"
-            }
-        ]
-    }
-    ```
+```
+docker run -p 3000:3000 --rm --name sourcebot -e CONFIG_PATH=sample-config.json ghcr.io/taqlaai/sourcebot:main
+```
 
-Sourcebot also supports indexing GitLab & BitBucket. Checkout the [index.json](./schemas/index.json) for a full list of available options.
+Navigate to `localhost:3000` in your favorite web browser to see Sourcebot running on your own machine! This example will index the Sourcebot repository and allow you to search through it. If you'd like to run Sourcebot on a different repository, please read [how to use Sourcebot on a custom repository](#using-sourcebot-on-a-custom-repository) !
 
-2. Create a Personal Access Token (PAT) to authenticate with a code host(s):
+<details>
+<summary>What does this command do?</summary>
 
-    <div>
-    <details open>
-    <summary>
-    <picture>
-        <source media="(prefers-color-scheme: dark)" srcset=".github/images/github-favicon-inverted.png">
-        <img src="https://github.com/favicon.ico" width="16" height="16" alt="GitHub icon">
-    </picture>
-    GitHub
-    </summary>
-    Generate a GitHub Personal Access Token (PAT) [here](https://github.com/settings/tokens/new). If you're only indexing public repositories select the `public_repo` scope; otherwise, select the `repo` scope.
+- Pull and run the Sourcebot docker image from [ghcr.io/taqlaai/sourcebot:main](ghcr.io/taqlaai/sourcebot:main)
+- Set the `CONFIG_PATH` environment variable in the container to `sample-config.json`. Sourcebot loads the config file located at `CONFIG_PATH` to determine which repositories to index. To make things easier to try Sourcebot, we've baked in an [example](https://github.com/TaqlaAI/sourcebot/blob/main/sample-config.json) config file named `sample-config.json` into the published Docker image. 
+- Map port 3000 between your machine and the docker image (`-p 3000:3000`). This is what allows you to run Sourcebot by navigating to `localhost:3000`
```

Comment:
nit: probably could just be `Map port 3000 between your machine and the docker image`